packages/framework-core/src/decorators/projects.ts

```diff
@@ -44,15 +60,46 @@ function registerProjection(
   projectionMetadata: ProjectionMetadata<EntityInterface, ReadModelInterface>
 ): void {
   Booster.configureCurrentEnv((config): void => {
-    const entityProjections = config.projections[originName] || []
-    if (entityProjections.indexOf(projectionMetadata) < 0) {
-      // Skip duplicate registrations
-      entityProjections.push(projectionMetadata)
-      config.projections[originName] = entityProjections
-    }
+    configure(config, originName, projectionMetadata, config.projections)
   })
 }
 
+function registerUnProjection(
+  originName: string,
+  projectionMetadata: ProjectionMetadata<EntityInterface, ReadModelInterface>
+): void {
+  Booster.configureCurrentEnv((config): void => {
+    configure(config, originName, projectionMetadata, config.unProjections)
+  })
+}
+
+function configure(
+  config: BoosterConfig,
+  originName: string,
+  projectionMetadata: ProjectionMetadata<EntityInterface, ReadModelInterface>,
+  configuration: Record<string, Array<ProjectionMetadata<EntityInterface, ReadModelInterface>>>
+): void {
+  const entityProjections = configuration[originName] || []
+  if (entityProjections.indexOf(projectionMetadata) < 0) {
+    // Skip duplicate registrations
+    entityProjections.push(projectionMetadata)
+    configuration[originName] = entityProjections
+  }
+}
```

Comment:
This function receives a param called ´config´ and another called `configuration`. It seems that the latter looks like the unprojection metadata. Could we rename this argument with a different name? (not sure what name should we use)

Reply:
Upon closer inspection, the `configuration` parameter is redundant since the projection configuration is already part of the `config` object. It was removed in commit 7b4b1e4.